index.html

```diff
@@ -56,6 +57,136 @@ <h1 class="describe">
+          <ul class="languages">
+            <li>Html</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="project-button-wrapper">
+            <a href="#project-modal" class="success-button">See Project</a>
+          </div>
+        </div>
+      </li>
+      <li class="card">
+        <div class="project-image">
+          <img class="projectimages" src="assets/images/project.png" alt="my project name">
+        </div>
+        <div class="project-description">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <ul class="languages">
+            <li>Html</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="project-button-wrapper">
+            <a href="#project-modal" class="success-button">See Project</a>
+          </div>
+        </div>
+      </li>
+      <li class="card">
+        <div class="project-image">
+          <img class="projectimages" src="assets/images/project.png" alt="my project name">
+        </div>
+        <div class="project-description">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <ul class="languages">
+            <li>Html</li>
+            <li>Css</li>
+            <li>JavaScript</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="project-button-wrapper">
+            <a href="#project-modal" class="success-button">See Project</a>
+          </div>
+        </div>
+      </li>
+    </ul>
+    <div id="project-modal" class="achievement-modal">
+      <div class="modal">
+        <a href="#" class="close-menu modal-close"><i class="fa fa-close"></i></a>
+        <img class="projectimages modal-banner" src="assets/images/project.png" alt="my project name">
+        <h2 class="modal-title">Keeping track of hundreds of components</h2>
+        <ul class="languages languages-left">
+          <li>Html</li>
+          <li>Css</li>
+          <li>JavaScript</li>
+          <li>JavaScript</li>
+        </ul>
+        <p>
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+          standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s
+          with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
+          the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it 1960s with the relea
+        </p>
+        <ul class="modal-buttons">
+          <li><a href="#" class="success-button medium-size">See Live <i class="fa fa-external-link"></i></a></li>
+          <li><a href="#" class="success-button medium-size">See Source <i class="fa fa-github"></i></a></li>
+        </ul>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] Great job implementing the whole "works" section. According to the requirements, you should also do the "about me" section, as you can see here:

![image](https://user-images.githubusercontent.com/66967274/143449850-6c91d3b9-a165-4b75-be7e-f6e175f2e700.png)